### DISCUSSION TITLE
commit-option for --amend

### DISCUSSION THREAD

**Discussion by imbobz** [stefanzweifel/git-auto-commit-action#337] [Help] [answered]
I just started to use this github action today, it is very useful, but i wanted to amend my commit instead of create an other.
By creating a "commit-option" we could do it :)

**Comment by imbobz** [comment, marked as answer]
didnt saw the commit-option in the doc ...

**Reply by stefanzweifel** [reply]
We also have some dedicated docs for `--amend`. Check them out: https://github.com/stefanzweifel/git-auto-commit-action?tab=readme-ov-file#using---amend-and---no-edit-as-commit-options
